### DISCUSSION TITLE
Has the recent switch to `blink.cmp` removed the `<C-y>` keybind for accepting suggestions?

### DISCUSSION THREAD

**Discussion by ThePaulMcBride** [LazyVim/LazyVim#5051] [Q&A] [answered]
I use `<C-y>` all the time for accepting suggestions, but after upgrading to LazyVim 14, it no longer works.

Is this an intentional removal or is it a bug?

**Comment by dpetka2001** [comment]
You want `opts.keymap.preset = "default"` or you can configure your own mappings for blink. Read its docs for more info. Or you can re-enable `nvim-cmp` from `:LazyExtras` if you want.

**Reply by ThePaulMcBride** [reply]
Is that not the default or does LazyVim change from that in some way?

**Reply by dpetka2001** [reply]
LazyVim uses the `enter` preset with some additional mappings for `<Tab>` and `<C-y`> that was added this morning. You can however choose to switch back to the `default` preset or just use entirely mappings of your own according to your own preferences. `blink`'s docs mention this.

**Comment by folke** [comment, marked as answer]
Good point. Just added that back. Will make a new release later today

**Reply by ThePaulMcBride** [reply]
Thank you, sir 🙏🏻
